Build/wcDocker.css

```diff
@@ -288,6 +292,7 @@ html, body {
 .wcLockFull .wcPanelTab,
 .wcPanelTab.wcNotMoveable {
   cursor: default;
+  background-color: unset !important;
```

Comment:
Just curious - why do we need this?

Reply:
Yes. Because when you hover on the title of floating window, the background of title becomes light blue, showing that it is dockable. With this, the background won't change.